docs/docs-components/data/components.js

```diff
@@ -495,32 +495,34 @@ const componentData: $ReadOnlyArray<ComponentData> = [
       },
     },
   },
-  // {
-  //   id: 'Chart',
-  //   platform: {
-  //     web: {
-  //       name: 'Chart',
-  //       visual: {
-  //         svg: <Chart />,
-  //       },
-  //       description: 'WIP ',
-  //       category: ['Data'],
-  //       status: {
-  //         accessible: {
-  //           summary: 'notAvailable',
-  //           a11yVisual: 'notAvailable',
-  //           a11yScreenreader: 'notAvailable',
-  //           a11yNavigation: 'notAvailable',
-  //           a11yComprehension: 'notAvailable',
-  //         },
-  //         documentation: 'notAvailable',
-  //         figmaStatus: 'notAvailable',
```

Comment:
We do have Figma components for bar (both horizontal and column) and line charts. But, they need to be renamed and updated with the latest to match code. I think we can mark as Ready here; designers can mak a note in Figma that there will be updates.